archive.php

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+</head>
+<body>
+	<?php
+	include 'navbar.php';
+	?>
+	<div id="container">
+		<script>
+		var youtube_videos = ['https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
```

Comment:
Made the Changes.

On Mon, Aug 31, 2015 at 9:45 AM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38331446>
> :
>
> > @@ -0,0 +1,31 @@
> > +<html>
> > +<head>
> > +<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
> > +<link rel="stylesheet" href="archive.css">
> > +<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
> > +</head>
> > +<body>
> > +	<?php
> > +	include 'navbar.php';
> > +	?>
> > +	<div id="container">
> > +		<script>
> > +		var youtube_videos = ['https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
>
> @hitesh4 <https://github.com/hitesh4> You don't need to include the
> repetitive string https://www.youtube.com/embed/ here. Just keep the
> video ID's in this array, and then add that string to the ID while
> generating the elements.
>
> **
>
> cc @students-alumni-cell-iitkgp/web-team-members
> <https://github.com/orgs/students-alumni-cell-iitkgp/teams/web-team-members>
> @aniarya82 <https://github.com/aniarya82>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38331446>
> .
>